.travis.yml

```diff
@@ -27,6 +27,8 @@ env:
   - TEST_TARGET="test-suite" COMPILER="4.02.3+32bit"
   - TEST_TARGET="validate"                           TW="travis_wait"
   - TEST_TARGET="validate"   COMPILER="4.02.3+32bit" TW="travis_wait"
+  - TEST_TARGET="ci-bedrock-src"                     TW="travis_wait 45"
```

Comment:
There's a file that takes 1000--1500 seconds.  Without travis_wait, Travis decides something is wrong after 600 seconds.